docs/design/specs/Ecma-335-Augments.md

```diff
@@ -399,7 +399,9 @@ The major changes are:
 
 * Interfaces are now allowed to have instance methods (both virtual and non-virtual). Previously we only allowed abstract virtual methods.
   * Interfaces obviously still can't have instance fields.
+  * Interfaces may also define static virtual methods (either as abstract or by directly providing the default implementation in the declaration).
```

Comment:
It looks like this is formatted as a sub-bullet under the "Interfaces are now allowed to have instance methods" bullet. Perhaps it should be on the same level instead.

Reply:
Thanks Aleksey for spotting, I originally did think it was intentional but I think you're right, especially considering the bullet list only has four items, the two-level structure doesn't make any sense.